src/FixedDataTableRow.js

```diff
@@ -463,6 +455,7 @@ class FixedDataTableRow extends React.Component {
           {...this.props}
           offsetTop={undefined}
           zIndex={undefined}
+          visible={undefined}
```

Comment:
no need to pass prop `visible`

Reply:
this is a funny way to remove props from this.props.  We might consider doing:
```
const {offsetTop, zIndex, visible, ...passedProps} = this.props;
...
<FDTRowImpl
    {...passedProps}
...
```

Reply:
yep, will do